### DISCUSSION TITLE
i can`t publish my project

### DISCUSSION THREAD

**Discussion by raaedmagdy** [twbs/bootstrap#38140] [General]
![Screenshot 2023-02-28 214128](https://user-images.githubusercontent.com/94644123/221961297-e7df3186-63a8-4b0d-b0b6-b44016417727.png)
i can`t publish my project when i use mdb publish in my command it get me a link when i open it it show my that page of bootstrap welcome

**Comment by ascensus-mdb** [comment]
Just to let you know - for support relating to MDB GO you should use this forum: https://mdbootstrap.com/support/

Bootstrap team cannot help you with it, because MDB GO is an external tool.

But regarding your problem - what's the name of the HTML file where your project is located? In the URL you should specify the path to this file, for example:

https://my-project.mdbgo.io/homepage.html 

The main link, like https://my-project.mdbgo.io/, will always refers to index.html and if you left index.html untouched, it will show its original content.

**Reply by raaedmagdy** [reply]
thanks for your help
